bookshelf/app.yaml

```diff
@@ -0,0 +1,12 @@
+runtime: custom
+vm: true
+
+manual_scaling:
```

Comment:
Most new users will be using free trial money.  I fear people will start with this config, and not realize they're running single instance with a poor uptime.  I'd rather start them off with a load balanced 2 instance setup, and let them scale down if they want to.